pxr/usdImaging/usdviewq/appController.py

```diff
@@ -2203,6 +2203,9 @@ def setFrameField(self, frame):
 
     # Prim/Attribute search functionality =====================================
 
+    def _normalize_unicode(self, str: str, form = 'NFKC'):
```

Comment:
It does make sense for usdview as a client to do this sort of normalization and not in the core USD, but we think it might still make sense to provide in brief in-code comments on why NFKC is being chosen for normalization.

Note that it was discussed with @mati-nvidia and others that at least for identifiers NFKC normalization should be discouraged, and encourage NFC normalization.